src/utils/generateIdentityToken.js

```diff
@@ -0,0 +1,19 @@
+const jwt = require('jsonwebtoken')
+
+const generateIdentityToken = (kind, uid, user) => {
```

Comment:
- I think we can keep `kind` hard-coded in the payload. Remove this parameter.
- I don't think we need `user`. Remove this parameter.